pkg/api/validation/validation.go

```diff
@@ -2766,8 +2766,10 @@ func ValidateAppArmorPodAnnotations(annotations map[string]string, spec *api.Pod
 			allErrs = append(allErrs, field.Invalid(fldPath.Key(k), containerName, "container not found"))
 		}
 
-		if err := apparmor.ValidateProfileFormat(p); err != nil {
-			allErrs = append(allErrs, field.Invalid(fldPath.Key(k), p, err.Error()))
+		if p != "unconfined" {
```

Comment:
`"unconfined"` should be defined as a constant, [here](https://github.com/kubernetes/kubernetes/blob/00c1ec52013e68ca7dcbd156ef898bd6d5104f3a/pkg/security/apparmor/helpers.go#L35)